scalafix-core/shared/src/main/scala/scalafix/internal/config/DisableUnlessConfig.scala

```diff
@@ -6,19 +6,34 @@ import org.langmeta.Symbol
 import scalafix.internal.config.MetaconfigPendingUpstream.XtensionConfScalafix
 import scalafix.internal.util._
 
+case class UnlessConfigSymbol(symbol: Symbol.Global, message: Option[String])
+
+object UnlessConfigSymbol {
+  implicit val decoder: ConfDecoder[UnlessConfigSymbol] =
+    ConfDecoder.instanceF[UnlessConfigSymbol] {
+      case c: Conf.Obj =>
+        (c.get[Symbol.Global]("symbol") |@|
+          c.getOption[String]("message")).map {
+          case (a, b) => UnlessConfigSymbol(a, b)
+        }
+      case els =>
+        implicitly[ConfDecoder[Symbol.Global]]
```

Comment:
Random thought not related to this PR, but adding an implicit apply on ConfDecoder would save these implicitly calls